services/metrics-calculator/Pipfile

```diff
@@ -7,29 +7,29 @@ name = "pypi"
 gp2gp-metrics-calculator = {editable = true, path = "."}
 
 [dev-packages]
-black = "==22.3.0"
-bandit = "~=1.7"
-flake8 = "~=3.9"
-flake8-bugbear="~=21.9"
-flake8-comprehensions="~=3.6"
-mccabe="~=0.6"
-flake8-builtins="~=1.5"
-flake8-print="~=4.0"
-isort = "~=5.9"
-mypy = "~=0.9"
-types-python-dateutil = "~=2.8"
+black = "==25.11.0"
+bandit = "~=1.9.2"
+flake8 = "~=7.2.0"
+flake8-bugbear="~=25.11.29"
+flake8-comprehensions="~=3.17.0"
+mccabe="~=0.7.0"
```

Comment:
I cannot find this referenced in any CVE checkers - it looks like it's a random person's complexity checking tool that hasn't been updated in almost 4 year. https://pypi.org/project/mccabe/0.7.0/